README.md

```diff
@@ -199,7 +199,7 @@ Use your preferred system for setting up a virtualenv, docker environment,
 or whatever else, then run the following:
 
 ```sh
-python -m pip install -e .[dev]
+python -m pip install -e '.[dev]'
```

Comment:
I'm running zsh as my shell and without the quotes, this gave me this error: `zsh: no matches found: .[dev]`